locales/en.json

```
@@ -1,6 +1,143 @@
+  "UserManagement": {
+    "title": "User Management",
+    "addNewUser": "Add New User",
+    "allUsers": "All Users",
+    "usersTotal": "users total",
+    "noName": "No Name",
+    "edit": "Edit",
+    "delete": "Delete",
+    "noUsersFound": "No users found. Create your first user!",
+    "createNewUser": "Create New User",
+    "editUser": "Edit User",
+    "userDetails": "User Details",
```

Biome (2.1.2)

[error] The key editUser was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)